app/views/pages/changes_999l.html.erb

```diff
@@ -0,0 +1,468 @@
, 2924297037, 2924297040, 2924297045, 2924297046, 2924297052, 2924297053, 2924297055, 2924297062, 2924297064, 2924297067, 2924297070, 2924297073, 2924297075, 2924297085, 2924297087, 2924297088, 2924297089, 2924297091, 2924297092, 2924297093, 2924297094, 2924297097, 2928009010, 2928009013, 2928009018, 2928009023, 2928009025, 2928009028, 2928009030, 2928009033, 2928009040, 2928009045, 2928009048, 2928009050, 2928009055, 2928009075, 2928009080, 2930100000, 2930909812, 2930909815, 2930909816, 2930909817, 2930909821, 2930909822, 2930909823, 2930909825, 2930909826, 2930909827, 2930909830, 2930909832, 2930909833, 2930909835, 2930909838, 2930909840, 2930909843, 2930909845, 2930909850, 2930909853, 2930909855, 2930909860, 2930909864, 2930909868, 2930909877, 2930909878, 2930909879, 2930909880, 2930909881, 2930909889, 2930909893, 2930909894, 2930909895, 2930909896, 2930909897, 2932990010, 2932990013, 2932990015, 2932990018, 2932990023, 2932990025, 2932990027, 2932990028, 2932990033, 2932990043, 2932990045, 2932990047, 2932990050, 2932990053, 2932990065, 2932990080, 2932990085, 2933360000, 2933370000, 2933399909, 2933399910, 2933399911, 2933399912, 2933399915, 2933399918, 2933399921, 2933399924, 2933399925, 2933399927, 2933399928, 2933399931, 2933399932, 2933399933, 2933399935, 2933399935, 2933399936, 2933399937, 2933399938, 2933399942, 2933399946, 2933399947, 2933399948, 2933399952, 2933399955, 2933399957, 2933399959, 2933399960, 2933399961, 2933399962, 2933399964, 2933399965, 2933399967, 2933399968, 2933399970, 2933399971, 2933399973, 2933399976, 2933399977, 2933399978, 2933399979, 2933399985, 2933499025, 2933499030, 2933499045, 2933499070, 2933540000, 2933599529, 2933920000, 2933998005, 2933998006, 2933998007, 2933998008, 2933998009, 2933998011, 2933998012, 2933998013, 2933998014, 2933998015, 2933998016, 2933998018, 2933998019, 2933998020, 2933998021, 2933998022, 2933998023, 2933998024, 2933998027, 2933998030, 2933998032, 2933998033, 2933998034, 2933998036, 2933998037, 2933998038, 2933998040, 2933998041, 2933998042, 2933998045, 2933998046, 2933998047, 2933998048, 2933998053, 2933998054, 2933998055, 2933998056, 2933998058, 2933998059, 2933998061, 2933998063, 2933998067, 2933998068, 2933998071, 2933998072, 2933998073, 2933998075, 2933998078, 2933998081, 2933998082, 2934999010, 2934999012, 2934999015, 2934999016, 2934999018, 2934999020, 2934999021, 2934999022, 2934999023, 2934999024, 2934999025, 2934999026, 2934999027, 2934999028, 2934999030, 2934999035, 2934999037, 2934999039, 2934999041, 2934999042, 2934999043, 2934999044, 2934999049, 2934999056, 2934999059, 2934999060, 2934999061, 2934999062, 2934999063, 2934999064, 2934999066, 2934999067, 2934999068, 2934999071, 2934999074, 2934999075, 2934999077, 2934999078, 2934999079, 2934999080, 2934999083, 2934999084, 2934999085, 2934999086, 2934999087, 2934999088, 2937230000, 2939630000, 2939791000, 2939799010, 2939799020, 2939799030, 2939799040, 2939799050, 2939799060, 3004410000, 3004420000, 3004430000, 3502909000
```

Comment:
these are data; should we hard code them in the view , or
maybe put them in YAML and read them via a controller? :thinking: 

not sure, it depends if  they never change

Reply:
Im not sure i think this is fine for now, its very satic content and will never know if they change or not as i guess policy. I think there is talk about presenting them in a different format so i think be wasted to mess around too much with them now. all these format changes were imagined by me and HMRC seem vaguely happy with it so will see what comes up.